models/coreModels/Email.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+
+const emailSchema = new mongoose.Schema({
+  emailKey: {
+    type: String,
+    unique: true,
+    lowercase: true,
+    required: true,
+  },
+  emailName: {
+    type: String,
+    unique:true,
+    required:true,
+  },
+  emailVariables:{
+    type:Array
```

Comment:
what's reason to use this emailVariables ?

for custom email variable , we can use pattern {{client.name}} , then when we send email , we can replace those variable before sending mail

Reply:
This is for two purpose , first is to let user know that these are the varables that you might use in that particular email and second  is to parse only those variables when sending the mail 
![image](https://github.com/idurar/erp-crm/assets/67551927/113d350e-9564-4cbf-bafb-6fbdf0f8dbed)

Reply:
perfect